node/network/bridge/src/multiplexer.rs

```diff
@@ -105,6 +110,17 @@ impl Stream for RequestMultiplexer {
 	}
 }
 
+impl FusedStream for RequestMultiplexer {
```

Comment:
can we remove this `.fuse()` now?
https://github.com/paritytech/polkadot/blob/4866f8e72029e719d7ce74e341bb1e94ebd724f7/node/network/bridge/src/lib.rs#L172